integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -68,11 +70,14 @@ def _partition_items(self, channel_metadata_item_map):
             content_id = item.content_id
             channel_metadata = item.channel_metadata
             transmitted_item = transmission_map.get(content_id, None)
+
             if transmitted_item is not None:
                 if diff(channel_metadata, transmitted_item.channel_metadata):
                     items_to_update[content_id] = channel_metadata
+                    items_catalog_updated_times[content_id] = item.catalog_last_changed
             else:
                 items_to_create[content_id] = channel_metadata
+                items_catalog_updated_times[content_id] = item.catalog_last_changed
```

Comment:
Is there a point in putting this line in the if block? You define it the same in both conditions.

Reply:
Also yeah what if there is no existing `catalog_last_changed` value? I know at some point we set it to `catalog_last_modified` or something which is also None by default (I assume it will take on a value though)

Reply:
the if block is supposed to determine if there needs to be an update or create, there's the case were it's neither (course already created and no updated needed) and in those instances I don't want to add the catalog last changed timestamps to the dict. Basically wherever we add things to either `items_to_update` or `items_to_create` (not `items_to_delete` as there's no timestamps saved for those, we're just deleting the record). 

If the `catalog_last_changed` is None at this point, then this had to have been from a CSOD transmission- basically it reads the payload for the `catalog_last_changed` value and the exporter creates the catalog_last_changed value from the request from ent-catalog. If there's a None value then it will simply save the transmissions without the the value. This is roughly the same as requiring the next transmission to update.